deepchecks/checks/methodology/single_feature_contribution_train_test.py

```diff
@@ -129,3 +172,33 @@ def condition(value: t.Dict[Hashable, float]) -> ConditionResult:
         return \
             self.add_condition(f'Train-Test features\' {pps_html_url} (PPS) difference is not greater than {threshold}',
                                condition)
+
+    def add_condition_feature_pps_in_train_not_greater_than(self: FC, threshold: float = 0.2) -> FC:
```

Comment:
```suggestion
    def add_condition_feature_pps_in_train_not_greater_than(self: FC, threshold: float = 0.85) -> FC:
```

Reply:
I think that's too high. Changing to 0.7, ok?